### ISSUE TITLE
[feature request] drone recall

### ISSUE THREAD

**Issue by Zygus42** [mcenderdragon/Futurepack-API#182]
Would love to see an option on the Drone Station to recall and shutdown the drone.  Finding it to change it's commands can be quite difficult if you have a large mining area.

**Comment by mcenderdragon** [comment]
Yeah we already plan to change the whole drone /drone station system.
